### PR TITLE
EASY-2144: set-title

### PR DESCRIPTION
**Pull request by stephanHogenboom**
fixes EASY-2144: set-title

#### When applied it will

- [x] merge [EASY-2143](https://github.com/DANS-KNAW/easy-springfield/pull/23) first

- [x] refactor after merge of 2143

#### related pull requests on github
repo                       | PR
-------------------------- | -----------------
easy-springfield                      | [PR#23](https://github.com/DANS-KNAW/easy-springfield/pull/23)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -302,6 +303,18 @@ ARGUMENTS
                                         before playing the presentation (private
                                         audio/video) or not (public audio/video)
         ---
+        
+        Subcommand: set-title - Sets the title of a video/audio within a presentation. The springfield-presentation-path
+                    |must be either a direct presentation referid or full path that can be resolved to a presentation.
```
Comment (rvanheest) [inline README.md]: having this `|` here doesn't seem correct...

File: src/main/scala/nl.knaw.dans.easy.springfield/CommandLineOptions.scala [hunk 2]
```diff
@@ -202,6 +203,23 @@ class CommandLineOptions(args: Array[String], config: Configuration) extends Sca
   }
   addSubcommand(setRequireTicket)
 
+  val setTitle = new Subcommand("set-title") {
+    descr("""Sets the title of a video/audio within a presentation. The springfield-presentation-path
+            |must be either a direct presentation referid or full path that can be resolved to a presentation.""")
```
Comment (rvanheest) [inline src/main/scala/nl.knaw.dans.easy.springfield/CommandLineOptions.scala]:
```suggestion
            |must be either a direct presentation referid or full path that can be resolved to a presentation.""".stripMargin)
```

File: src/main/scala/nl.knaw.dans.easy.springfield/CommandLineOptions.scala [hunk 3]
```diff
@@ -203,6 +204,24 @@ class CommandLineOptions(args: Array[String], config: Configuration) extends Sca
...
+  val setTitle = new Subcommand("set-title") {
+    descr(
+      """Sets the title of a video/audio within a presentation. The springfield-presentation-path must be either a direct presentation referid
+        | or full path that can be resolved to a presentation.""".stripMargin)
+  val title: ScallopOption[String] = opt(name = "title", short = 't',
+    descr = "The (new) name of the video/audio element with a presentation",
+    required = true)
+    val videoNumber: ScallopOption[String] = opt(name = "video-number", short = 'v',
+      descr = "The number/ index of the video within the presentation",
+      default = Some("1")) //TODO default 1 or require and fail if not provided?
```
Comment (janvanmansum) [inline src/main/scala/nl.knaw.dans.easy.springfield/CommandLineOptions.scala]: Remove TODO. The default is fine.

Comment (janvanmansum) [inline src/main/scala/nl.knaw.dans.easy.springfield/CommandLineOptions.scala]:
```suggestion
      default = Some("1")) 
```

**Pull request merged**